CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.0.5] - 2020-07-13
```

Comment:
```suggestion

```
We only add the version when we do a release, like this one for example: https://github.com/scanapi/scanapi/pull/192. For now, we can leave everything under [Unreleased].